backend/server.js

```diff
@@ -9,19 +9,29 @@ import {
 } from "./src/routes/index.js";
 import express from "express";
 import cookieParser from "cookie-parser";
+import { fileURLToPath } from "url";
+import path from "path";
+
 const app = express();
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// middleware
 app.use(cookieParser());
+app.use(express.static(path.join(__dirname, "../frontend/public")));
+app.use(express.static(path.join(__dirname, "../frontend/node_modules")));
 
 const port = process.env.PORT || 3000;
 
 // routes
-app.use("/", loginRouter);
-app.use("/", productRouter);
-app.use("/", logoutRouter);
-app.use("/", userRouter);
-app.use("/", salesRouter);
-app.use("/", saleDetailsRouter);
-app.use("/", cartRouter);
+app.use("/api/v1", loginRouter);
+app.use("/api/v1/products", productRouter);
+app.use("/api/v1/logout", logoutRouter);
+app.use("/api/v1", userRouter);
+app.use("/api/v1/sales", salesRouter);
+app.use("/api/v1/sale-details", saleDetailsRouter);
```

Comment:
I dont think you need a separate salesDetail route. Sales should be able to handle that. it should just be `/sales/{id}/details`. Also currently how are you getting the sales details, no salesId or something?

Reply:
Also just checked the project requirements, it should be 3 sales endpoint

1.` GET /sales` to get all sales only by admin
2. `GET /sales/<saleId>` to get a particular sale record and should only be accessible by the admin and the attendant that created the sales.
3. `POST /sales` to create sale and it is only accessible to a store attendant.